### PR TITLE
check if mono is running with --debug arg

### PR DESCRIPTION
**Pull request by margaale**
#### Database Migration
NO

#### Description
I Haven't test on a linux box. See issue #480

### CODE REVIEW

File: src/NzbDrone.Core/HealthCheck/Checks/MonoDebugCheck.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        private readonly Logger _logger;
+        private readonly StackFrameHelper _stackFrameHelper;
+
+        public MonoDebugCheck(Logger logger, StackFrameHelper stackFrameHelper)
+        {
+            _logger = logger;
+            _stackFrameHelper = stackFrameHelper;
+        }
+
+        public class StackFrameHelper
```
Comment (margaale) [inline src/NzbDrone.Core/HealthCheck/Checks/MonoDebugCheck.cs]: Where shoud i put this class? I had to create it just for mocking purpose...

**Comment by Taloth** [comment]
@markus101 Maybe we should do this only for new installs? Or just once during startup and log a warn instead of a health check.
I'm just not sure about it all coz ppl can't dismiss health checks.

**Comment by markus101** [comment]
A warning during start would work. We can always add a health check later once we support dismissing them.

**Comment by Taloth** [comment]
K, then i'll have this healthcheck log a warn, but return ok.

**Comment by Taloth** [comment]
Merged via https://github.com/Sonarr/Sonarr/commit/13a259b47313100b53dfacd4423107da19a2cf9d

**Pull request closed without merging**